spog/ui/src/console/mod.rs

```diff
@@ -88,6 +88,17 @@ fn authenticated_page(props: &ChildrenProperties) -> Html {
 
     let callback_github = use_open("https://github.com/trustification/trustification", "_blank");
 
+    let url_str;
+    match &config.global.expanded_page_url{
+        Some(url) => {
+            url_str = url.as_str();
+        }
+        None =>{
+            url_str = "";
+        }
+    };
+    let callback_expanded_page = use_open(url_str, "_blank");
```

Comment:
The `use_open` hook might not work here, as it's not conditional. I think you might need to use `use_callback`, and check inside if the `url` is something, or not. And only then open the page.